src/middleware/errorHandlingMiddleware.ts

```diff
@@ -0,0 +1,14 @@
+import { NextFunction, Request, Response } from "express";
+import { HttpException } from "../utils/exceptions/httpException";
+ 
+export function errorHandlingMiddleware(error: HttpException, request: Request,
+                                        response: Response, next: NextFunction) {
+  const status = error.status || 500;
+  const message = error.message || "Something went wrong";
```

Comment:
Improve the error message by adding information about the error.

Reply:
By default the error handling middleware should return:
HTTP-Code`500` and  `error.message`
**nothing** else.

We also need to discuss why error can only be of type `HttpException`.